templates/compute_pulse_heights.ipynb

```diff
@@ -2,139 +2,37 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import sys 
</pre><p>Wow, this notebook got a major marie kondo makeover. Looks much better! Now it's time to mirror the formatting of the other notebooks, explanations of what this is for, etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>